### ISSUE TITLE
opts={'caption':"Windows Title Showed?"} The Caption  NOT show in windows?

### ISSUE THREAD

**Issue by dragen1860** [facebookresearch/visdom#369]
Hi, I have several windows showing different images, and I need the `caption` property showing in windows title bar to recognise each other. However,
```python
vis.image(img, win='img', opts={'caption':"Windows Title Showed?"})

```
When i set the opts.caption parameters, the caption dont appear as :
![image](https://user-images.githubusercontent.com/4252555/41146016-82efc30e-6b34-11e8-8996-277b3088a712.png)

Am I wrong?

**Comment by lvdmaaten** [comment]
If you drag the window to be bigger than the image, does the caption appear?

**Comment by dragen1860** [comment]
@lvdmaaten thx, it does not appear .

**Comment by dragen1860** [comment]
I found the solution, should set `title` key instead of `caption`.
```python
vis.image(img, win='img', opts={'title':"Windows Title Showed?"})
```
![image](https://user-images.githubusercontent.com/4252555/41219657-8ff6cd40-6d91-11e8-9dad-120d08a425e0.png)
I dnt know what does `caption` means yet.

**Comment by JackUrb** [comment]
Caption _is_ supposed to show some text underneath the image rather than above it (as a title) but for some reason that isn't working right now for you. Will investigate further on my end.

**Comment by pmwenzel** [comment]
The caption option is still not working with visdom version '0.1.8.5'. Do you have any suggestion on how to fix this issue? Thanks.

**Comment by JackUrb** [comment]
There is a fix for captions in #393, however #393 introduces issues that I need to fix before pulling it in.
